### ISSUE TITLE
contentEncoding set to base64 is generated as JsonString instead of JsonBase64String

### ISSUE THREAD

**Issue by challe** [corvus-dotnet/Corvus.JsonSchema#526]
I'm currently trying to upgrade from version 3.0.27 to 4.1.3 of the type generator tool.
Here's a stripped down version of a schema that we use.

```json
{
  "$schema": "https://json-schema.org/draft/2020-12/schema",
  "type": "object",
  "properties": {
    "file": {
      "type": "object",
      "properties": {
        "content": {
          "type": "string",
          "contentEncoding": "base64",
          "contentMediaType": "application/octet-stream"
        }
      }
    }
  }
}
```

Before upgrading, the File.Content type was `Corvus.Json.JsonBase64String` which allowed me to use `GetDecodedBase64Bytes()`.

After the upgrade, it looks like this type is now set to `Corvus.Json.JsonString`.

**Comment by mwadams** [comment]
In V3 we ignored unknown `contentMediaType` declaration. In V4 we are more finicky about recognizing content media types before applying the format.

I will add `application/octet-stream` as a "known-type" for base64 encoded-string support.

**Comment by mwadams** [comment]
When the CI/CD has got through, this fix will be available in V4.1.4
